### PR TITLE
Disallow empty tags

### PR DESCRIPTION
**Pull request by nelsonkopliku**
Makes the backend a bit more rebust when creating tags.

Frontend couterpart has not been addressed here.

### CODE REVIEW

File: lib/trento_web/controllers/tags_controller.ex [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  defp resource_type_from_path("hosts"), do: {:ok, :host}
+  defp resource_type_from_path("clusters"), do: {:ok, :cluster}
+  defp resource_type_from_path("sap_systems"), do: {:ok, :sap_system}
+  defp resource_type_from_path("databases"), do: {:ok, :database}
+  defp resource_type_from_path(_), do: {:error, :unrecognized_resource}
+
+  defp empty?(<<" "::binary, rest::binary>>), do: empty?(rest)
+  defp empty?(""), do: true
+  defp empty?(_string), do: false
+
```
Comment (fabriziosestito) [inline lib/trento_web/controllers/tags_controller.ex:56]: not sure what this does? couldn't you check the length of the string?

Comment (nelsonkopliku) [reply lib/trento_web/controllers/tags_controller.ex:56]: this guy checks for empty and blank strings :smile:

**Review by dottorblaster — APPROVED** [review]
LGTM

**Pull request merged**